homeassistant/components/tesla_wall_connector/sensor.py

```diff
@@ -127,13 +127,21 @@ class WallConnectorSensorDescription(
         entity_category=EntityCategory.DIAGNOSTIC,
     ),
     WallConnectorSensorDescription(
-        key="energy_kWh",
+        key="energy_Wh",
```

Comment:
Please move this change to a separate PR fixing the bug, we should not mix it with a PR adding new features.